spiped: 1.5.0 -> 1.6.1

birlorg opened the pull request:
upgrade spiped from 1.5.0 to 1.6.1

This is my first PR to nix, so please let me know if I messed something up!



<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change

upgrade spiped from 1.5.0 to 1.6.1, which is the new stable version per: https://www.tarsnap.com/spiped.html

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ x ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

birlorg committed:
spiped: 1.5.0 -> 1.6.1

upgrade spiped from 1.5.0 to 1.6.1

r-rmcgibbo commented:
Result of `nixpkgs-review pr 122237` at 7e419992 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/f22af68cffb3e96a74454f5a207aaebf">spiped</a></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/tools/networking/spiped/default.nix:29:3:
```
   |
29 |   installPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/patch-phase.md">patch-phase</a>

`patchPhase` should not be overridden, use `postPatch` instead.

Near pkgs/tools/networking/spiped/default.nix:14:3:
```
   |
14 |   patchPhase = ''
   |   ^
```
</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

birlorg committed:
spiped: 1.5.0 -> 1.6.1

fix changes per @r-rmcgibbo

birlorg committed:
spiped: 1.5.0 -> 1.6.1

oops, I'm editing using the web UI and I forgot to delete this patch as it's no longer needed.

birlorg commented:
@r-rmcgibbo  I'm pretty sure I fixed everything up now.  can you test again?

rmcgibbo commented:
Looks like there's an eval problem with ofborg

rmcgibbo commented on pkgs/tools/networking/spiped/default.nix:
```diff
@@ -2,33 +2,30 @@
-      --replace "rm" "${coreutils}/bin/rm"   \
-      --replace ">/dev/stderr" ">stderr.log" \
-      --replace "2>/dev/null" "2>stderr.log"
-  '';
```
I think you didn't mean to remove the closing quotes and semicolon here.

rmcgibbo replied:
Also, after you fix this it would be great if you could squash the commits into 1 commit.

birlorg replied:
You are correct, Sorry.  I have it working locally, but I don't have a way to push from my nixos machine into Github land, so I'm editing the file directly in the Github UI, which is less than ideal, and leads to these typos like this.  I guess I should have copy/pasted the entire file, instead of trying to surgically edit.

It appears to squash commits, requires me to be able to push from a local git clone, which I'm unable to do.  Unless you know of a way to do it in the GitHub Web UI?

Anyways, this latest commit puts back the '';

birlorg committed:
spipe: 1.5.0 -> 1.6.1

fix by re-adding '';

thoughtpolice reviewed (approved):
lgtm
